### DISCUSSION TITLE
Breaking changes in v1 release?

### DISCUSSION THREAD

**Discussion by zchenyu** [helmfile/helmfile#1497] [Questions]
We have a Github action that runs helmfile that started failing today:

```
in ./helmfile.yaml: failed to read helmfile.yaml: reading document at index 1: [3:4] unexpected mapping key
       2 | releases:
    >  3 |   {{- if has .Environment.Name (list "default" "prod") }}
              ^
       4 |   - name: prod
```

I suspect it's due to this v1 release: https://github.com/helmfile/helmfile/releases/tag/v1.0.0-rc.0

Is there any documentation around what breaking changes were made and what we need to do to fix them?
For now I'll see if I can just pin to an older version

**Comment by yxxhero** [comment]
You can use v0.164.0. I will see what happened.

**Comment by yxxhero** [comment]
@zchenyu please  rename your file with .gotmpl suffix.

**Reply by mumoshu** [reply]
@zchenyu Also see [this section of the v1 proposal](https://github.com/helmfile/helmfile/blob/main/docs/proposals/towards-1.0.md#force-gotmpl-file-extension-for-helmfileyaml-in-case-you-want-helmfile-to-render-it-as-a-go-template-before-yaml-parsing)! You can start using the .gotmpl suffix in v0.x today then you need no change when upgrading to v1.
